examples/draw_turtle.rs

```diff
@@ -0,0 +1,323 @@
+//! https://arkada38.github.io/2018/01/14/creating-a-turtle/
+//!
+//! Inspired by child's drawing of turtle we are going to draw and paint figures
+//! which consists of arcs and straight lines.
+//! To draw arcs we need to use the loops which are determine tilt angle and the length.
+//! The more sophisticated the figure is the more loops it requires to make.
```

Comment:
This is great! This clarifies the technique used throughout this code very well. :smile: 

Since I know we both want this explanation to be as understandable as possible, I'm going to help out by making a few adjustments to your grammar.

```rust
//! This is inspired by a child's drawing of a turtle. We are going to draw the turtle using
//! arcs and straight lines.
//! To draw arcs, we use multiple Rust for-loops to create different tilt angles and
//! lengths. The more sophisticated the figure is, the more loops we need to make it.
```

I think this comment accurately expresses what you said. Feel free to make changes as you see fit. :smile: